lib/menu.rb

```diff
@@ -0,0 +1,15 @@
+
+class Menu
+    attr_reader :menu 
```

Comment:
No need for this attribute reader given that the view method below is doing exactly the same thing.